docs/quickstart.md

```diff
@@ -1,6 +1,6 @@
 # Getting started
 
-## pulp-insta-demo.sh
+## Insta demo
```

Comment:
Is this "instant", or "installation"?

Reply:
I don't know, in every place I saw "insta" https://pulpproject.org/insta-demo/
@mikedep333 may know what it means

Reply:
Or "Instance"?

Reply:
"instant". A common marketing prefix in English.